agents-common/src/main/java/org/apache/ranger/plugin/service/RangerBasePlugin.java

```diff
@@ -86,6 +87,7 @@ public RangerBasePlugin(String serviceType, String serviceName, String appId) {
 	}
 
 	public RangerBasePlugin(RangerPluginConfig pluginConfig) {
+		this.pluginAnalytics = new RangerPluginAnalytics();
```

Comment:
RangerBasePlugin has 6  constructor, "pluginAnalytics "  instance variable is not getting initialised from all 6 code paths.  Please do let me know if i am looking the wrong code base.

Reply:
I see just 4 constructors - line 511, 515, 519 and 565 - in RangerBasePlugin.java
@vyommani Can you point to the other 2 constructors you are seeing ?
Constructor 1: Always calls constructor 3
Constructor 2: Always calls constructor 4
So I added pluginAnalytics function calls only for constructor 3 and constructor 4. 
Adding it to all the 4 constructors would result in double counting